internal/accounts/contract-add.go

```diff
@@ -66,8 +67,11 @@ func addContract(
 	if err != nil {
 		return nil, err
 	}
-
-	account, err := services.Accounts.AddContract(to, name, code, false)
+	contractArgs := state.Config().Deployments.ByContractNameAndNetwork(name, globalFlags.Network).Args
+	if contractArgs == nil {
+		contractArgs = []cadence.Value{}
```

Comment:
These shouldn't be fetched from the config, they should be passed as command arguments the way that's done for transactions for example https://github.com/onflow/flow-cli/blob/master/internal/transactions/send.go#L36-L37